Sources/StreamVideoSwiftUI/ViewFactory.swift

```diff
@@ -153,6 +153,13 @@ public protocol ViewFactory: AnyObject {
         callSettings: Binding<CallSettings>,
         call: Call?
     ) -> LocalParticipantViewModifierType
+
+    associatedtype UserAvatarViewType: View
+    func makeUserAvatar(
+        _ user: User,
+        size: CGFloat,
+        failBackProvider: (() -> AnyView)?
+    ) -> UserAvatarViewType
```

Comment:
Would be nice to have an "Options" paremeter as well to follow the same API as the Chat SDK: https://github.com/GetStream/stream-chat-swiftui/pull/734